From 6999022f3b1c62894ade0002ab32a9212ce908f9 Mon Sep 17 00:00:00 2001
From: Philip Withnall <philip.withnall@collabora.co.uk>
Date: Mon, 11 Jul 2016 10:27:35 -0700
Subject: [PATCH] connman.service: Use ProtectSystem=true to allow writing
 /etc/localtime

Setting the timezone requires unlinking and relinking /etc/localtime, so
we need /etc to be mounted read-write. This means that commit dc8f151e
has to be softened to ProtectSystem=true rather than
ProtectSystem=full. This mounts most of the filesystem as read-only,
apart from /etc, which is read-write.
---
 src/connman.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/connman.service.in b/src/connman.service.in
index 7376346..a485414 100644
--- a/src/connman.service.in
+++ b/src/connman.service.in
@@ -15,7 +15,7 @@ ExecStart=@sbindir@/connmand -n
 StandardOutput=null
 CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_BIND_SERVICE CAP_NET_RAW CAP_SYS_TIME CAP_SYS_MODULE CAP_SYS_ADMIN
 ProtectHome=true
-ProtectSystem=full
+ProtectSystem=true
 
 [Install]
 WantedBy=multi-user.target
-- 
2.1.4

